CHANGELOG.md

```diff
@@ -5,6 +5,57 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] - October, 31, 2023
+
+### React
+
+#### Breaking
+
+#### Added
+
+- [Footer] Added onLogoClick callback
+- [Container] Added a new attribute to align it with Header
+- [Login] Added a system to handle login and api tokens
+
+#### Fixed
+
```

Comment:
Missing information about info, invalid and success feedbacks to input components from [this PR.](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1150)

Reply:
CHANGELOG is now updated